exercises/matching-brackets/example.jl

```diff
@@ -0,0 +1,33 @@
+function matching_brackets(a)
+	a = split(a, "")
+	b,c = ["(", "[", "{"], [")", "]", "}"]
+	a = filter(i -> (i in b || i in c), a)
```

Comment:
I'm fine with it as it is, but you could make it slightly more readable by using `i in b ∪ c` or `i in union(b, c)`